protocol.md

```diff
@@ -0,0 +1,196 @@
+Status Secure And Decentralized Messaging Protocol
+==================================================
+
+# Abstract
+
+Ethereum empowers users and developers to interact with totally new kind of applications called Dapps (Decentralized applications). These application allows to interact with the blockchain on a completely new level which is not only about exchanging values but also executing arbitrary logic. This logic can form very sophisticated programs like DAOs (Decentralized autonomous organizations). The missing part here is how users of Dapps can communicate securely and in a decentralized way with each other. Communication is an essential part of any activity. In this document, we specify a secure and decentralized messaging protocol that is capable of running on the Ethereum network.
+
+# Introduction
+
+TBD
+
+# Terminology
+
+* *Client*: a Whisper node implementing the protocol
+* *Whisper node*: an Ethereum node with Whisper V6 enabled (in the case of geth, it's `--shh` option)
```

Comment:
Somewhere it should mention Geth as req due to Parity incompatiblity (with link to implementation in Parity and Nimbus)